semcore/notice-bubble/src/style/notice-bubble.shadow.css

```diff
@@ -55,6 +55,11 @@ SNoticeBubble {
     width: calc(100% - var(--intergalactic-spacing-6x, 24px));
   }
 }
+SNoticeBubble[disablePortal] {
+  position: sticky;
+  margin-left: auto;
```

Comment:
Why did you used `margin-left` instead of `right`?

Reply:
Because we need to "crop" the notice from the left side by content. If don't - it will be 100% width.